definitions/contested/json/CaseEvent/CaseEvent.json

```diff
@@ -765,5 +765,19 @@
     "SecurityClassification": "Public",
     "ShowEventNotes": "N",
     "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "13/12/2019",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "FR_consentOrderApproved",
+    "Name": "Consent Application Approved",
+    "Description": "Consent Application Approved",
+    "DisplayOrder": 116,
+    "PreConditionState(s)": "*",
```

Comment:
this is to be changed to 'Awaiting Judiciary Response (consent)' when https://tools.hmcts.net/jira/browse/FR-1746 is merged